README.md

```diff
@@ -113,6 +114,6 @@ I would like to thank [Microverse](https://www.microverse.org/) for this opportu
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] Great effort working on some of the changes requested by the previous reviewer, But currently the requested changes about the broken MIT license link is not implemented yet the link is still throwing `error 404`. I notice the name of the file you targetted for the license should be `[MIT](./MIT.md)` and not `[MIT](./MIT)`

MIT License 👇 

<img width="571" alt="mit" src="https://github.com/wolterbwambale/vet-clinic/assets/78634841/27a86941-66ee-4bfc-8ad3-97242a76137d">